tests/queries/0_stateless/02676_format_decimal.sql

```diff
@@ -0,0 +1,20 @@
+-- Regular types
+SELECT formatDecimal(2, 2);  -- more digits required than exist
+SELECT formatDecimal(2.123456, 2);  -- rounding
```

Comment:
Actually, the docs don't say that the fractional digits are potentially rounded (it sounds like they are truncated). Clarify that?